manifests/config_nginx.pp

```diff
@@ -18,9 +18,7 @@
 
   package {
     'nginx':
-      ensure => installed,
-      before => Exec['Chown graphite for web user'],
-      notify => Exec['Chown graphite for web user'];
```

Comment:
If you switch from apache to nginx weeks later for example, or if you install the nginx config in a next iteration, it will run chown a second time. And still there is no relationship between the nginx package and the permissions on the storage dir, even the nginx vhost doesn't reference it at all (only gunicorn and the apache vhost do), so it's not needed.